src/styles/header.scss

```diff
@@ -0,0 +1,73 @@
+@import "./product-card-styles/variables.scss";
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  padding: 0 50px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
you can create variables with these lines move this logic into `mixins.scss` and reuse in this place